src/main/java/org/assertj/core/api/Assertions.java

```diff
@@ -136,7 +136,7 @@
  * @author William Delanoue
  */
 @CheckReturnValue
-public class Assertions {
+public class Assertions implements InstanceOfAssertFactories {
```

Comment:
let's also make `BDDAssertions` implement `InstanceOfAssertFactories`

Reply:
`BDDAssertions` extends `Assertions` so it's already getting the factories, do you want the `implements` to be explicit also here?